theia-full-docker/Dockerfile

```diff
@@ -80,36 +80,35 @@ RUN apt-get -y install git sudo
 #LSPs
 
 ##GO
-ENV GO_VERSION 1.11.4
+ENV GO_VERSION 1.12
 ENV GOPATH=/usr/local/go-packages
 ENV GO_ROOT=/usr/local/go
 ENV PATH $PATH:/usr/local/go/bin
 ENV PATH $PATH:${GOPATH}/bin
 
-RUN curl -sS https://storage.googleapis.com/golang/go$GO_VERSION.linux-amd64.tar.gz | tar -C /usr/local -xzf - && \
-    go get -u -v github.com/nsf/gocode && \
-    go get -u -v github.com/uudashr/gopkgs/cmd/gopkgs && \
-    go get -u -v github.com/ramya-rao-a/go-outline && \
-    go get -u -v github.com/acroca/go-symbols && \
-    go get -u -v golang.org/x/tools/cmd/guru && \
-    go get -u -v golang.org/x/tools/cmd/gorename && \
-    go get -u -v github.com/fatih/gomodifytags && \
-    go get -u -v github.com/haya14busa/goplay/cmd/goplay && \
-    go get -u -v github.com/josharian/impl && \
-    go get -u -v github.com/tylerb/gotype-live && \
-    go get -u -v github.com/rogpeppe/godef && \
-    # godoc causing issues ATM, preventing image to build - commenting-out
-    # go get -u -v golang.org/x/tools/cmd/godoc && \
-    go get -u -v github.com/zmb3/gogetdoc && \
-    go get -u -v golang.org/x/tools/cmd/goimports && \
-    go get -u -v sourcegraph.com/sqs/goreturns && \
-    go get -u -v github.com/golang/lint/golint && \
-    go get -u -v github.com/cweill/gotests/... && \
-    go get -u -v github.com/alecthomas/gometalinter && \
-    go get -u -v honnef.co/go/tools/... && \
-    go get -u -v github.com/sourcegraph/go-langserver && \
-    go get -u -v github.com/derekparker/delve/cmd/dlv && \
-    go get -u -v github.com/davidrjenni/reftools/cmd/fillstruct
+RUN curl -sS https://storage.googleapis.com/golang/go$GO_VERSION.linux-amd64.tar.gz | tar -C /usr/local -xzf -
+RUN go get -u -v github.com/nsf/gocode
+RUN go get -u -v github.com/uudashr/gopkgs/cmd/gopkgs
+RUN go get -u -v github.com/ramya-rao-a/go-outline
+RUN go get -u -v github.com/acroca/go-symbols
+RUN go get -u -v golang.org/x/tools/cmd/guru
+RUN go get -u -v golang.org/x/tools/cmd/gorename
+RUN go get -u -v github.com/fatih/gomodifytags
+RUN go get -u -v github.com/haya14busa/goplay/cmd/goplay
+RUN go get -u -v github.com/josharian/impl
+RUN go get -u -v github.com/tylerb/gotype-live
+RUN go get -u -v github.com/rogpeppe/godef
+RUN go get -u -v github.com/zmb3/gogetdoc
+RUN go get -u -v golang.org/x/tools/cmd/goimports
+RUN go get -u -v sourcegraph.com/sqs/goreturns
+# RUN go get -u -v github.com/golang/lint/golint
+RUN go get -u -v github.com/cweill/gotests/...
+RUN go get -u -v github.com/alecthomas/gometalinter
+RUN go get -u -v honnef.co/go/tools/...
+# RUN go get -u -v github.com/sourcegraph/go-langserver
+RUN go get -u -v github.com/go-delve/delve/cmd/dlv
+RUN go get -u -v github.com/davidrjenni/reftools/cmd/fillstruct
+RUN go get -u -v github.com/godoctor/godoctor
```

Comment:
Do we really want to create a new docker leader for each dependency? This will increase the final size of the image.

Reply:
> Do we really want to create a new docker leader for each dependency? This will increase the final size of the image.

No, it's just for test purposes. I kept the pull-request as a draft until I was happy with it.

Reply:
Got it! I commented too early... sorry :)

Reply:
> Got it! I commented too early... sorry :)

No worries! :)